apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/TrygdetidService.kt

```diff
@@ -313,7 +319,10 @@ class TrygdetidServiceImpl(
                     ident = avdoedMedFnr.first,
                     yrkesskade = false,
                 )
-            val opprettetTrygdetid = trygdetidRepository.opprettTrygdetid(trygdetid)
+            val pesystt = pesysKlient.hentTrygdetidsgrunnlag(avdoedMedFnr.first, brukerTokenInfo)
```

Comment:
Som tidligere nevnt så er jeg ikke helt overbevist om at det er en god ide å bare dunke inn disse periodene sånn uten videre her av følgende argumenter:
- Nedetid i Pesys vil føre til nedetid på trygdetid i Gjenny
- Blir det tydelig for saksbehandler at disse periodene magisk kom fra Pesys? Er det kilde som sier noe om dette?
- Ønsker man alltid å få inn disse periodene?
- Usikker på om dette er OBO eller ikke, men hvis så, hva om saksbehandler ikke har tilgang til pesys? 

Kunne man evnt latt saksbehandler hente inn perioder som finnes i pesys med en egen en knapp? Litt som man får valget om man ønsker å kopiere trygdetid fra annen sak på samme avdød.

Reply:
Svar fra Mari:

- For å svare på det siste først: Ja, jeg ser for meg at sb kunne hentet tt fra AP/UT med å klikke på en knapp. Og så må de kunne redigere.
- Ja, kilde kan vise at periodene kommer fra sak i pesys
- Så sant det er registrert tt på avdød i AP/UT, er dette fint å få inn i alle sakene det gjelder
- Som sagt må man kunne redigere periodene, og når det gjelder fremtidig tt (vil bare være i UT-saker) kan vi opplyse der at dette må sb legge inn selv i disse sakene, dersom det er vanskelig å få inn denne